test/e2e_test.go

```diff
@@ -1179,16 +1179,16 @@ func TestTlog(t *testing.T) {
 	// Now verify should work!
 	must(verify(pubKeyPath, imgName, true, nil, ""), t)
 
-	// Now we turn on the tlog!
-	defer setenv(t, env.VariableExperimental.String(), "1")()
+	// TODO: priyawadhwa@ to figure out how to add an entry to the tlog without using keyless signing
```

Comment:
I'm hesitant about removing tests - I think these should be a part of the PR and not separate.